### ISSUE TITLE
Support for custom vGPU resource names

### ISSUE THREAD

**Issue by cgapu** [volcano-sh/volcano#3925]
### What is the problem you're trying to solve

I am using Volcano and the Project-HAMi/volcano-vgpu-device-plugin plugin for GPU resource virtualization. This plugin supports modifying vGPU-related resources, but in Volcano, these resources are hardcoded. The relevant resource names are:
 - volcano.sh/vgpu-number
 - volcano.sh/vgpu-memory
 - volcano.sh/vgpu-cores"

### Describe the solution you'd like

 I hope to expose them as startup parameters so that users can customize them.

**Comment by Monokaix** [comment]
@archlitchi Please check this: )

**Comment by archlitchi** [comment]
yes, i plan to use a CM to store corresponding parameters, we'll discuss this at weekly meeting
